src/test/kotlin/no/skatteetaten/aurora/boober/feature/AuroraAzureApimSubPartTest.kt

```diff
@@ -57,40 +58,42 @@ class AuroraAzureApimSubPartTest : AbstractMultiFeatureTest() {
     }
 
     @Test
-    fun `if disabled policies are not taken into consideration`() {
+    fun `if policies are added and sorted`() {
         val (_, auroraAzureApp) = generateResources(
             """{
              "azure" : {
                 "apim": {
                     "enabled"    : true,
-                    "path"       : "/unique",      
-                    "openapiUrl" : "https://openapi",
-                    "serviceUrl" : "https://service",
-                    "policies"   : {"justDisable":false, "disabledToo": false},  
-                    "apiHost"    : "api.apimanagement.dev.skatteetaten.io"   
+                    "apiName"    : "super-api",
+                    "version"    : "v1", 
+                    "path"       : "path/to/api",
+                    "openApiURL" : "https://openapi",
+                    "serviceURL" : "https://service",
+                    "policies"   : [ "policy2", "policy1" ]
```

Comment:
Ser endringen fra `policy1=true` til bare `policy`. Jeg mener å huske at @bentsolheim hadde en innvending her om at dette vil gjøre det vanskelig å overstyre en config slik ta den blir deaktivert.

Altså om du på / definerer policy1=true, så kan du under /utv04/ definere policy1=false for å fjerne den. Dette klarer du ikke om du bare bruker "policy1".

Reply:
Jeg lager en ny revisjon for hver eneste endring av API'et. Ellers synes jeg det er litt for risikabelt. Det blir jo dessuten (med mindre jeg hele tiden differ mot APIM) vanskelig å se forskjell på policy=false, og fraværet av en policy. Derfor tok jeg vekk flagget.

Men defaults-greiene hadde jeg ikke tenkt på. Jeg synes egentlig slike defaults ikke er så lurt, men når vi først har det, må det jo være mulig å slå av ting...

Men kanskje det er bedre å lage et objekt med en gang? Altså:

    policy1:
       enabled: true

Da støtter vi utvidelser, for eksempel om det blir behov for å sette spesielle indekser, rate-limiting, ip-filtre etc. (ikke et behov enda, men da har vi muligheten).

Hva tenker dere om det siste forslaget? @nostra @bentsolheim ?

Reply:
Jeg overlater til @bentsolheim å svare på denne.

Reply:
Enig der, men det er jo greit om config-formatet støtter det. For eksempel er det krav om eget format på feilmeldinger, som skal inneholder en bestemt feilkode. Den er forskjellig fra hvert API, og hører kanskje hjemme i en policy.

Reply:
Ja, dette med å skru ting av og på er lettere dersom man benytter en map fremfor en liste. Dersom du ønsker å endre på hvilke policies som er aktive i et gitt miljø for en gitt app, så må du overstyre hele listen med en ny komplett liste for det gitte miljøet. Med en map kan man skru av enkelt-policies. 

Dette er jo kun en relevant betraktning dersom det er en aktuell problemstilling. Hvis det f.eks. sjeldent skjer, eller at det "er like greit", evt. tydeligere, å bare overstyre hele listen, har jeg ingen innvendinger.

Når det gjelder policies kommer jeg til kort. Skal policiene være navngitte referanser til policies som er deklarert "et annet sted", eller skal policiene og egenskapene dem deklareres i AuroraConfig?

Reply:
@bentsolheim I første omgang skal det være navngitte policies som finnes i Mudbunny. Men jeg ser for meg at det veldig raskt kommer behov for litt tweaking, og formatet med et eget enablet-felt gjør det jo enkelt å endre senere. På bekostning av bittelitt mer auroraconfig.